sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_greengrass_example.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    export BUCKET='sagemaker-eu-west-1-699391019698'
</pre><p>Don't leave your own bucket hardcoded. You can use the SageMaker default bucket variable that is already initialized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>